CONTRIBUTING.md

```diff
@@ -1,81 +1,49 @@
 
-Before you can do anything, you first need a [GitHub account](https://github.com/signup/free). This is required because we use GitHub to handle all incoming *Pull Requests* (code modifications) and *Issues* (bug reports) which cannot be made without a GitHub account.
+## Make Your First Commit
 
+What makes the DuckDuckGo community so great is how quickly new contributors can start committing code. To see just how quickly we mean, try our [Quick Start Goodie](https://duck.co/duckduckhack/goodie_quickstart) Tutorial. It will help you set up your environment, fork the repository, and walk you through creating the simplest possible Instant Answer.
 
-## Submitting a **Bug** or **Suggestion**
+From there, you could learn more about the [various Instant Answer types](https://duck.co/duckduckhack/determine_your_instant_answer_type), or get hacking on a real Instant Answer, below.
 
-- Firstly, please make sure the bug is related to the **Spice** repository. If this bug is about the DuckDuckGo API, or the relevancy of our search results, please visit our feedback page at <https://duckduckgo.com/feedback>. If you're unsure, its best to use the feedback page (your message will be passed along to the correct people).
+If you have any questions at any point, feel free to ask on one of our community channels:
 
-- Check the **Spice** [issues](https://github.com/duckduckgo/zeroclickinfo-spice/issues) to see if an issue already exists for the given bug or suggestion
-  - If one doesn't exist, create a GitHub issue in the **Spice** repository
-    - Clearly describe the bug/improvemnt, including steps to reproduce when it is a bug
-  - If one already exists, please add any additional comments you have regarding the matter
+*   [DuckDuckHack mailing list](https://duck.co/redir/?u=https%3A%2F%2Fwww.listbox.com%2Fsubscribe%2F%3Flist_id%3D197814)
+*   [Gitter chat](https://duck.co/redir/?u=https%3A%2F%2Fgitter.im%2Fduckduckgo%2FIdeas)
 
-If you're submitting a **pull request** (bugfix/addition):
-- Fork the **Spice** repository on GitHub
+Do not hesitate to email us directly at [open@duckduckgo.com](mailto:open@duckduckgo.com).
 
+## Improve a Live Instant Answer
 
-## Making Changes
+We recommend that your first live contribution to Instant Answers is to improve one that's already live. Many contributors report completing their first fix within two hours of forking the repository.
 
-- Before making any changes, refer to the [DuckDuckHack Styleguide](https://duck.co/duckduckhack/styleguide_overview) to ensure your changes are made in the correct fashion
-- Make sure your commits are of a reasonable size. They shouldn't be too big (or too small)
-- Make sure your commit messages effectively explain what changes have been made, and please identify which instant answer or file has been modified:
+The best place to get started is by looking at "low-hanging fruit" fixes, which we've made sure to identify throughout our repositories for new contributors.
```

Comment:
Can we link "low-hanging fruit" to https://github.com/duckduckgo/zeroclickinfo-spice/issues?q=is%3Aopen+is%3Aissue+label%3A%22Low-Hanging+Fruit%22 ?